src/test/java/org/kohsuke/github/ArchTests.java

```diff
@@ -0,0 +1,56 @@
+package org.kohsuke.github;
+
+import com.tngtech.archunit.core.domain.JavaClass;
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.domain.properties.HasAnnotations;
+import com.tngtech.archunit.core.domain.properties.HasName.AndFullName;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.lang.ArchCondition;
+import com.tngtech.archunit.lang.ArchRule;
+import com.tngtech.archunit.lang.ConditionEvents;
+import com.tngtech.archunit.lang.SimpleConditionEvent;
+import org.junit.Test;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+
+public class ArchTests {
+
+    private final JavaClasses classFiles = new ClassFileImporter()
```

Comment:
"classFiles" could be a static variable.

Then I would add this:

``
    @BeforeClass
    public static void beforeClass() {
        assertTrue(classFiles.size() > 0);
    }
```

Reply:
@sullis 
Which way do you think it better?